src/App.tsx

```diff
@@ -11,157 +14,20 @@ export const App = () => (
       aria-label="main navigation"
     >
       <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
+        <div className={'navbar-brand'}>
+          <NavLink className={getLinkClass} to="/">
```

Comment:
You can move all paths in enums or constants and use it in all places in your application.